.golangci.yml

```diff
@@ -8,31 +8,43 @@ linters:
         - ineffassign
         - staticcheck
         - stylecheck
-        - typecheck
```

Comment:
why type check is removed? we have it `jarvis-proto`

Reply:
Good question. I removed it, because I don't see it here anymore: https://golangci-lint.run/usage/linters/

But if it is still there, maybe it is issue in documentation. Will double check

Reply:
Putting back, thanks